Presentation.Web/app/shared/navigation/navigation-authorized.view.html

```diff
@@ -1,20 +1,20 @@
 ﻿<div class="col-xs-8">
     <ul class="nav navbar-nav">
         <li>
-            <a data-ui-sref="organization.overview" data-ng-class="{'active': $state.includes('organization')}" data-ng-if="user.currentConfig.showTabOverview">Organisation</a>
-            <a data-ui-sref="organization.structure" data-ng-class="{'active': $state.includes('organization')}" data-ng-if="!user.currentConfig.showTabOverview">Organisation</a>
+            <a data-ui-sref="organization.overview" data-ng-class="{'active': $state.includes('organization')}" data-ng-if="user.currentConfig.showTabOverview" data-hook="organizationButton">Organisation </a>
+            <a data-ui-sref="organization.structure" data-ng-class="{'active': $state.includes('organization')}" data-ng-if="!user.currentConfig.showTabOverview" data-hook="organizationButton">Organisation</a>
         </li>
         <li data-ng-if="user.currentConfig.showItProjectModule">
-            <a data-ui-sref="it-project.overview" data-ng-class="{'active': $state.includes('it-project')}"><span ng-if="user.currentConfig.showItProjectPrefix">IT </span>Projekter</a>
+            <a data-ui-sref="it-project.overview" data-ng-class="{'active': $state.includes('it-project')}" data-hook="projectButton"><span ng-if="user.currentConfig.showItProjectPrefix">IT </span>Projekter</a>
```

Comment:
Hvorfor kalder du dem data-hook? Skulle det ikke være data-element-type. Se evt. spec for data hooks i HTML5: http://html5doctor.com/html5-custom-data-attributes/
De behøves ikke hedde data-hoo